api/core/helper/code_executor/python3/python3_transformer.py

```diff
@@ -33,7 +33,7 @@
 
 PYTHON_STANDARD_PACKAGES = {
     'json', 'datetime', 'math', 'random', 're', 'string', 'sys', 'time', 'traceback', 'uuid', 'os', 'base64',
-    'hashlib', 'hmac', 'binascii', 'collections', 'functools', 'operator', 'itertools', 'uuid',
+    'hashlib', 'hmac', 'binascii', 'collections', 'functools', 'operator', 'itertools',
```

Comment:
Is this an irrelevant change?

Reply:
No, this is auto-fixed by the rule B033 `duplicate-value` (Checks for set literals that contain duplicate items.).
`uuid` is duplicated in PYTHON_STANDARD_PACKAGES on line 35

Reply:
Okay, I saw it, thx.